### ISSUE TITLE
New term: Erythematous migratory rash 

### ISSUE THREAD

**Issue by jpazmandi** [obophenotype/human-phenotype-ontology#6204]
**Preferred term label:**
Erythematous migratory rash  

**Synonyms**
Red, blistering rash that spreads across the skin.

**Definition (free text, please give PubMed ID)**
In comparison to Erythema migrans (HP:0031180), which is an expanding rash specifically seen in the early stage of Lyme disease; erythematous migratory rash is a migratory rash not specific to Lyme disease, and it is a phenotype of the following diseases: TRAPS MIM: 142680,  as explained in https://pubmed.ncbi.nlm.nih.gov/27894451/. 

**Parent term (use hpo.jax.org/app)**
HP:0010783 (Erythema)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
142680 (MIM phenotype ID)

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
I do not think that blistering is correct, this is a different phenotypic feature of this class of disease.
I am adding the following term
Label: Migratory erythematous plaque
A migratory, centrifugal, erythematous, tender, non-purpuric, and well-demarcated plaque. This feature may be observed in TNF receptor-associated periodic syndrome, in which it often occurs together with migratory myalgia in muscles located underneath the affected areas of skin. PMID:31736939
